test/e2e/framework/ingress_utils.go

```diff
@@ -372,6 +372,19 @@ func CleanupGCEIngressController(gceController *GCEIngressController) {
 	}
 }
 
+func (cont *GCEIngressController) ListGlobalForwardingRules() []*compute.ForwardingRule {
+	gceCloud := cont.Cloud.Provider.(*gcecloud.GCECloud)
+	fwdList := []*compute.ForwardingRule{}
+	l, err := gceCloud.ListGlobalForwardingRules()
+	Expect(err).NotTo(HaveOccurred())
+	for _, fwd := range l {
+		if cont.canDelete(fwd.Name, fwd.CreationTimestamp, false) {
```

Comment:
canDelete seems to be a strange name; should this actually be isOwned?

Reply:
Done. Added an isOwned function which is just a wrapper for canDelete.